backend/django_backend/feed/models.py

```diff
@@ -9,7 +9,7 @@ def validate_file_size(file):
 
 
 def validate_video_file_type(file):
-    valid_mime_types = ['video/mp4', 'video/mpeg']
+    valid_mime_types = ['video/mp4', 'video/mpeg', 'video/quicktime', 'video/x-msvideo', 'video/x-ms-wmv', 'video/webm', 'video/3gpp', 'video/3gpp2', 'video/x-matroska', 'video/ogg']
```

Comment:
Noen av disse som er mindre sikre og utsatt for angrep? 

Fks: video/x-msvideo (AVI)